diff --git a/requests/help.py b/requests/help.py
index cd961c78..5440ee61 100644
--- a/requests/help.py
+++ b/requests/help.py
@@ -1,23 +1,24 @@
 """Module containing bug report helper(s)."""
 from __future__ import print_function
 
 import json
 import platform
 import sys
 import ssl
 
+import idna
 import urllib3
 import chardet
 
 from . import __version__ as requests_version
 
 try:
     from .packages.urllib3.contrib import pyopenssl
 except ImportError:
     pyopenssl = None
     OpenSSL = None
     cryptography = None
 else:
     import OpenSSL
     import cryptography
 
@@ -72,44 +73,48 @@ def info():
     urllib3_info = {'version': urllib3.__version__}
     chardet_info = {'version': chardet.__version__}
 
     pyopenssl_info = {
         'version': None,
         'openssl_version': '',
     }
     if OpenSSL:
         pyopenssl_info = {
             'version': OpenSSL.__version__,
             'openssl_version': '%x' % OpenSSL.SSL.OPENSSL_VERSION_NUMBER,
         }
     cryptography_info = {
         'version': getattr(cryptography, '__version__', ''),
     }
+    idna_info = {
+        'version': getattr(idna, '__version__', ''),
+    }
 
     # OPENSSL_VERSION_NUMBER doesn't exist in the Python 2.6 ssl module.
     system_ssl = getattr(ssl, 'OPENSSL_VERSION_NUMBER', None)
     system_ssl_info = {
         'version': '%x' % system_ssl if system_ssl is not None else ''
     }
 
     return {
         'platform': platform_info,
         'implementation': implementation_info,
         'system_ssl': system_ssl_info,
         'using_pyopenssl': pyopenssl is not None,
         'pyOpenSSL': pyopenssl_info,
         'urllib3': urllib3_info,
         'chardet': chardet_info,
         'cryptography': cryptography_info,
+        'idna': idna_info,
         'requests': {
             'version': requests_version,
         },
     }
 
 
 def main():
     """Pretty-print the bug information as JSON."""
     print(json.dumps(info(), sort_keys=True, indent=2))
 
 
 if __name__ == '__main__':
     main()
