diff --git a/requests/models.py b/requests/models.py
index 752c58c1..caf13d66 100644
--- a/requests/models.py
+++ b/requests/models.py
@@ -30,36 +30,35 @@ from .utils import (
     iter_slices, guess_json_utf, super_len, to_native_string)
 from .compat import (
     cookielib, urlunparse, urlsplit, urlencode, str, bytes, StringIO,
     is_py2, chardet, json, builtin_str, basestring)
 from .status_codes import codes
 
 #: The set of HTTP status codes that indicate an automatically
 #: processable redirect.
 REDIRECT_STATI = (
     codes.moved,              # 301
     codes.found,              # 302
     codes.other,              # 303
     codes.temporary_redirect, # 307
     codes.permanent_redirect, # 308
 )
+
 DEFAULT_REDIRECT_LIMIT = 30
 CONTENT_CHUNK_SIZE = 10 * 1024
 ITER_CHUNK_SIZE = 512
 
-json_dumps = json.dumps
-
 
 class RequestEncodingMixin(object):
     @property
     def path_url(self):
         """Build the path URL to use."""
 
         url = []
 
         p = urlsplit(self.url)
 
         path = p.path
         if not path:
             path = '/'
 
         url.append(path)
@@ -413,31 +412,31 @@ class PreparedRequest(RequestEncodingMixin, RequestHooksMixin):
             self.headers = CaseInsensitiveDict()
 
     def prepare_body(self, data, files, json=None):
         """Prepares the given HTTP body data."""
 
         # Check if file, fo, generator, iterator.
         # If not, run through normal process.
 
         # Nottin' on you.
         body = None
         content_type = None
         length = None
 
         if json is not None:
             content_type = 'application/json'
-            body = json_dumps(json)
+            body = json.dumps(json)
 
         is_stream = all([
             hasattr(data, '__iter__'),
             not isinstance(data, (basestring, list, tuple, dict))
         ])
 
         try:
             length = super_len(data)
         except (TypeError, AttributeError, UnsupportedOperation):
             length = None
 
         if is_stream:
             body = data
 
             if files:
