diff --git a/requests/models.py b/requests/models.py
index 099f1c66..f9a2f5ab 100644
--- a/requests/models.py
+++ b/requests/models.py
@@ -359,21 +359,21 @@ class Response(object):
     def __repr__(self):
         return '<Response [%s]>' % (self.status_code)
 
 
     def __nonzero__(self):
         """Returns true if :attr:`status_code` is 'OK'."""
         return not self.error
 
 
     def raise_for_status(self):
-        """Raises stored :class:`HTTPError`, if one occured."""
+        """Raises stored :class:`HTTPError` or :class:`URLError`, if one occured."""
         if self.error:
             raise self.error
 
     def read(self, *args):
         """Returns :attr:`content`. Used for file-like object compatiblity."""
 
         return self.content
 
 
 
